sympy/stats/joint_rv_types.py

```diff
@@ -263,8 +300,7 @@ class MultivariateBetaDistribution(JointDistribution):
     _argnames = ['alpha']
     is_Continuous = True
 
-    @staticmethod
-    def check(alpha):
+    def check(self, alpha):
```

Comment:
Why did these definitions change?
These were decorated with `staticmethod` to match the definitions in `crv_types`, `drv_types` and `frv_types` in #16858.